scripts/install_bazel.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+#
+# This script installs Bazel for Debian or MacOS for usage in the Travis CI environment.
+#
+# Required Configs (from Travis-CI);
+# - CI
+# - TRAVIS
+# - TRAVIS_OS_NAME
+#
+# Optional configs:
+# - BAZEL_VERSION: Bazel version to install, default: 0.11.1
```

Comment:
If this file is Travis specific now, I would avoid defaulting to a particular Bazel version, just to make updates to Bazel simpler here. Instead, simply fail if BAZEL_VERSION is not specified.